src/Sylius/Bundle/ShippingBundle/Form/Type/ShippingMethodType.php

```diff
@@ -137,6 +145,13 @@ public function buildView(FormView $view, FormInterface $form, array $options):
         }
     }
 
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        parent::configureOptions($resolver);
+
+        $resolver->setDefault('rules_help', 'sylius.form.shipping_method.rules_help');
```

Comment:
I have no idea. I have added it directly to the template now. The intention is the same as the form option `help`, yes. But I don't think Sylius have implemented this yet? That's why I put it here, because I like having help/documentation directly where you need it.